### PR TITLE
Rver cancels subscriptions on correct gall ducts and cancels behn timers correctly as well

### PR DESCRIPTION
**Pull request by tacryt-socryp**

### CODE REVIEW

**Review by eglaysher — APPROVED** [review]

File: sys/vane/rver.hoon [hunk 1]
```diff
@@ -60,7 +60,7 @@
       $:  %b
           ::
           ::
-          $%  [%wake ~]
+          $%  [%wake error=(unit tang)]
```
Comment (joemfb) [inline sys/vane/rver.hoon]: This is just half of the necessary timer fix. A `%wake` where `error` is `~` is a normal timer, but when `error` is present it's a report of a previously-failed timer event. The fully correct way to handle this is to unwind whatever state was associated with that timer (maybe canceling subscriptions, builds, producing an HTTP 500 response etc.). The less correct approach is to just print the error via `%dill` if present (this is what most vanes are doing right now). But it's not correct to treat such an event as a normal timer wakeup.

**Review by joemfb — CHANGES_REQUESTED** [review]
The timer handling needs some tweaks, details below:

**Comment by joemfb** [comment]
contained-in/superceded-by #1202

**Comment by loganallenc** [comment]
Closing this as it is superseded by the Modulo merge PR.

**Pull request closed without merging**